tree/treeplayer/test/data.h

```diff
@@ -3,13 +3,26 @@
 
 #include <vector>
 struct Data {
-   unsigned int fUSize;
-   int fSize;
-   double* fArray; //[fSize]
-   float* fUArray; //[fUSize]
-   std::vector<double> fVec;
-   Double32_t fDouble32;
-   Float16_t fFloat16;
+   unsigned int fUSize{2};
+   int fSize{4};
+   double *fArray; //[fSize]
+   float *fUArray; //[fUSize]
+   std::vector<double> fVec{17., 18., 19., 20., 21., 22.};
+   Double32_t fDouble32{17.};
+   Float16_t fFloat16{44.};
+
+   Data() : fArray(new double[4]{12., 13., 14., 15.}), fUArray(new float[2]{42., 43.}) {}
```

Comment:
We should set distinct values in the default constructor and the constructor (or setter) used for Writing.  This allows to distinguish the case where the data was not read (in the right place) from the correctly working case.

Reply:
How is it done? (My reading is that there is only one constructor and that the values are not changed before calling fill).

Reply:
Well the class it's simple and it's used in such a way that only one constructor is enough, hence the default constructor. If you want I can create a separate user-defined constructor, without changing the rest of the test (i.e. without changing the currently written values).

Reply:
Changed in latest commit 👍